client/signup/steps/domains/index.jsx

```diff
@@ -632,6 +668,10 @@ class DomainsStep extends Component {
 		}
 
 		if ( isReskinned ) {
+			if ( ! stepSectionName && this.isExperiment ) {
```

Comment:
This should be a function call :) 
```suggestion
			if ( ! stepSectionName && this.isExperiment() ) {
```